File: python/raindrops/raindrops.py
def convert(number):
    value = get_raindrop_sound(number)

    return value if value else str(number)


def get_raindrop_sound(number):
    returnValue = ""

    if number % 3 == 0:
        returnValue += "Pling"

    if number % 5 == 0:
        returnValue += "Plang"

    if number % 7 == 0:
        returnValue += "Plong"

    return returnValue

File: javascript/raindrops/raindrops.js
export const convert = (raindrop) => {
  let sound = raindrop % 3 == 0 
    ? "Pling"
    : "";

  if (raindrop % 5 == 0)
    sound += "Plang";

  if (raindrop % 7 == 0)
    sound += "Plong";

  return sound 
    ? sound
    : raindrop.toString();
};
